/*const getData = async () =>
    await fetch(recipeArray)
      .then((res) => res.json())
      .catch((error) => console.log("erreur"));
*/